### DISCUSSION TITLE
Welcome to Owl Discussions!

### DISCUSSION THREAD

**Discussion by zethon** [zethon/Owl#110] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SamuraiCrow** [comment]
Hello! I am Samuel D. Crow and became interested in Owl as a means to making forums usable on platforms like the Commodore Amiga whose browser technology only made it as far as HTML3 or transitional4 and just barely that. I couldn't care less about modern platforms that have browsers already.
